pennylane/templates/state_preparations/state_prep_mps.py

```diff
@@ -15,24 +15,100 @@
 Contains the MPSPrep template.
 """
 
+import numpy as np
+
 import pennylane as qml
 from pennylane.operation import Operation
 from pennylane.wires import Wires
 
 
+def right_canonicalize_mps(mps):
+    """Transform an MPS into a right-canonical MPS.
+
+    Args:
+        mps (list[TensorLike]): List of tensors representing the MPS.
+
+    Returns:
+        A list of tensors representing the MPS in right-canonical form.
+
+    .. seealso:: :class:`~.MPSPrep`.
```

Comment:
Do we want an example? Or it's not necessary?

Reply:
MPS is a rather large structure and no small example is well visualized. Do you think it is worth to put an example anyway?

Reply:
No, I think it is fine personally since people can look at the example in MPSPrep to get a sense of how it's used.

Reply:
User-facing functions need to have and example. @DSGuala can make the final decision here.

Reply:
Example added 👍